### ISSUE TITLE
railway=tram stronger than light_rail at zoom 12

### ISSUE THREAD

**Issue by jojo4u** [openstreetmap-carto/openstreetmap-carto#1831]
Example light_rail: https://www.openstreetmap.org/way/316529558
Example tram: https://www.openstreetmap.org/way/316384343 and https://www.openstreetmap.org/way/30015494
At zoom 12 railway=tram ist stronger than railway=light_rail: https://www.openstreetmap.org/#map=12/48.7532/9.2163 this is also affecting the "new" style: https://osm.rrze.fau.de/map-ll-osm?zoom=12&layer=RRZE%20tileserver%20osm.org%20style%20tiles&lat=48.7572&lon=9.1922
Other zooms: 
At zoom 13, light_rail is getting much stronger, will file new issue for that.
At zoom 11, tram is not painted (not a problem).
#1828 and #1829 are related.

**Comment by kocio-pl** [comment]
Do you think you could make a test code for this?

**Comment by imagico** [comment]
After #5290 this is clearly no more the case so closing as fixed.
